docs/salesforce/release-notes.mdx

```diff
@@ -21,6 +21,18 @@ import {
 
 # Release Notes
 
+## 3.8
+
+_September 22, 2022_
+
+** Release Notes **
+
+- Case comments for closed cases can now be archived. Admins will see an option to archive data in 
+  - Helpshift Admin (LWC) -> Settings -> Archive data.
+  - This will help in reducing storage for Helpshift Cases in Salesforce.
+- Added fixes for clearing Salesforce security review, thus improving package security.
```

Comment:
Do we need to mention Salesforce security review?
We can just say - Add fixes for improving package security.